src/eos-updater-fetch.h

```diff
@@ -23,8 +23,8 @@
 #pragma once
 
 #include "eos-updater-generated.h"
-#include "eos-updater-util.h"
-#include <ostree.h>
+
+#include <gio/gio.h>
```

Comment:
Are we relying on `EosUpdater`'s declaration to be pulled in by `eos-updater-generated.h`? Perhaps it would be better to forward-declare it here.